.spectral.yaml

```diff
@@ -97,3 +97,23 @@ rules:
     then:
       field: type
       function: truthy
+
+  items-must-have-a-type:
+    description: "All items must have a type."
+    recommended: true
+    severity: warn
+    given: $.components.schemas..items
+    then:
+      field: type
+      function: truthy
+
+  application-json-response-must-not-be-type-string:
```

Comment:
Would be nice to document why by adding a comment above the rule or by improving the description